fvtr/speccpu2017/at-sniff32.cfg

```diff
@@ -0,0 +1,74 @@
+#  Copyright 2018 IBM Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+tune            = base
+output_format   = asc,cfg
+teeout          = 1
+log_line_width  = 156
+iterations      = 1
+reportable      = 0
+check_md5       = 0
+runlist         = all
+action          = validate
+backup_config   = 0
+
+makeflags       = %{MAKEFLAGS}
+
+CC = %{CC}
+CXX = %{CXX}
+FC = %{FC}
+
+# How to say "Show me your version, please"
+CC_VERSION_OPTION       = -v
+CXX_VERSION_OPTION      = -v
+FC_VERSION_OPTION       = -v
+
+
+#--------- Portability --------------------------------------------------------
+default:               # data model applies to all benchmarks
+EXTRA_PORTABILITY = -DSPEC_LP64
```

Comment:
LP64 is usually related to 64-bit machines.
What is `SPEC_LP64` doing in this case for 32-bit machines.